pycbc/dq.py

```diff
@@ -210,24 +222,18 @@ def query_flag(ifo, segment_name, start_time, end_time,
                 flag_segments += (partial.coalesce() & send)
 
         else:  # Standard case just query directly
-            try:
-                segs = query(':'.join([ifo, segment_name]),
-                             int(start_time), int(end_time),
-                             host=server)['active']
-                for rseg in segs:
-                    flag_segments.append(segment(rseg[0], rseg[1]))
-            except Exception as e:
-                print("Could not query flag, check name "
-                      "(%s) or times" % segment_name)
-                raise e
+            segs = query_dqsegdb2(ifo, segment_name, start_time, end_time,
+                                  server)
+            for rseg in segs:
+                flag_segments.append(segment(rseg[0], rseg[1]))
 
         # dqsegdb output is not guaranteed to lie entirely within start
         # and end times, hence restrict to this range
         flag_segments = flag_segments.coalesce() & \
             segmentlist([segment(int(start_time), int(end_time))])
 
     else:
-        raise ValueError("Source must be dqsegdb or GWOSC."
```

Comment:
Why are you changing this? "any" is the default value that checks both GWOSC and dqsegdb?

Reply:
Right, `any` is one of the three valid values one can pass, so I was just making the error message slightly more correct.